packages/components/pie-cookie-banner/src/index.ts

```diff
@@ -86,6 +99,65 @@ export class PieCookieBanner extends LitElement implements CookieBannerProps {
         this._isModalOpen = true;
     };
 
+    /**
+     * Handles the logic of the toggle switch nodes (preferences).
+     * Clicking the “all” toggle switch should turn on all preferences.
+     * When the “all” toggle is checked, and one of the other preferences is clicked,
+     * then the “all” toggle should be unchecked.
+     */
+    private _handlePreferencesToggled = (e: CustomEvent) : void => {
```

Comment:
Not sure why, but this name is a bit awkward to read.

I'd go with `handleToggleStates` or something like that.

Reply:
I tried to match the naming convention of the other handler “handlePrefercnesSaved” 😂 but that will work too, will update!